terraso_backend/apps/graphql/schema/sites.py

```diff
@@ -95,6 +97,9 @@ def resolve_seen(self, info):
             return True
         return self.seen_by.filter(id=user.id).exists()
 
+    def resolve_site_notes(self):
```

Comment:
Issue: Also this should be handled automatically if adding it to the fields array work